spec/controllers/collection_show_controller_spec.rb

```diff
@@ -37,5 +37,15 @@
     end
   end
 
-
+  describe "Bad params" do
+    let(:collection) { create(:collection, friendlier_id: "faked") }
+    it "doesn't throw an error when facet.page is an array" do
+      get :facet, params: {
+        id:              "subject_facet",
+        collection_id:   collection.friendlier_id,
+        "facet.page" =>  { goat: 'goat' }
```

Comment:
the title says "is an array", the actual test is a _hash_.  Which do you mean to test? Make the label match the thing?  Testing an array might look like `"facet.page" => ["one", "two"]`, no?